### PR TITLE
Created an SVG figure

### LINKED ISSUE #26923: Create an SVG figure for the MDoF Pendulum example
This example:

https://docs.sympy.org/dev/tutorials/physics/mechanics/multi_degree_freedom_holonomic_system.html

has a PNG as the figure:

image

I drew it on a Microsoft surface for a lecture a long time back and the surface gave wavy lines. This figure needs to be replaced with an SVG (pngs shouldn't be committed to the repo, this one was) and the figure should look better.

### PR DESCRIPTION
**Pull request by ZiYao54**
<Created a SVG figure >

#### References to other Issues or PRs
<Fixes #26923 >

#### Brief description of what is fixed or changed
Replaced the PNG file (multidof-holonomic.png) with a SVG file (multidof-holonomic.svg) drew by myself.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by moorepants** [comment]
Looks good:
image

**Comment by ZiYao54** [comment]
> Looks good: image

What should I do now? I'm confussed...

**Comment by moorepants** [comment]
This looks good to me. Thank you very much.

File: .mailmap [hunk 1]
```diff
@@ -1577,6 +1579,9 @@ Zhi-Qiang Zhou <zzq_890709@hotmail.com> zhouzq-thu <zzq_890709@hotmail.com>
 Zhongshi <zj495@nyu.edu>
 Zhuoyuan Li <zy.li@stu.pku.edu.cn> zylipku <123136644+zylipku@users.noreply.github.com>
 Zhuoyuan Li <zy.li@stu.pku.edu.cn> zylipku <zy.li@stu.pku.edu.cn>
+ZiYao <151983618+ZiYao54@users.noreply.github.com>
+ZiYao54 <ziyaoqzy2001@gmail.com>
+ZiYao54 <ziyaoqzy2001@gmail.com>
```
[the remark this answers is not in the document]
Comment (moorepants) [reply .mailmap:1584]: Should the mailmap be edited in a new PR then?

[the remark this answers is not in the document]
Comment (oscarbenjamin) [reply .mailmap:1584]: Yes. All the added lines should begin with the name and email address that is to be added in the AUTHORS file.

Probably what is needed is to delete all the lines that were added here and instead add these two lines:
```
Quek Zi Yao <ziyaoqzy2001@gmail.com> ZiYao <151983618+ZiYao54@users.noreply.github.com>
Quek Zi Yao <ziyaoqzy2001@gmail.com> ZiYao54 <ziyaoqzy2001@gmail.com>
```

[the remark this answers is not in the document]
Comment (moorepants) [reply .mailmap:1584]: @ZiYao54 can you open a new PR with this suggested fix?

[the remark this answers is not in the document]
Comment (ZiYao54) [reply .mailmap:1584]: Apologies for the mistakes. I've opened a new PR and fixed the mailmap file. Please review it here: #27168.

**Pull request merged**